### PR TITLE
Fix #1946: Optimise the code in drag_drop_single_item.xml

### LINKED ISSUE #1946: Optimise code in drag_drop_single_item.xml
This issue is tracking the optimisation work of `drag_drop_single_item.xml`

1. Make sure that all textviews use the style from `styles.xml` file like `Heading1`, `TextFieldLabel`, `Body`, etc. whichever suits best.

While making the above changes the UI should not get affected and the end result should still be same as before (or an optimised version of UI).

If there are multiple files with the same name, update them all.

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes #1946 , Used custom style "Subtitle1" to optimise drag_drop_single_item.xml

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.
@rt4914

### CODE REVIEW

**Comment by Arjupta** [comment]
@rt4914  PTAL

**Comment by aggarwalpulkit596** [comment]
Could you add a screenshot for before and after the change. @ARJUPTA

**Comment by Arjupta** [comment]
> Could you add a screenshot for before and after the change. @ARJUPTA 

I ended up finding its usage in test files. Can you help me find any other visible usage.

**Comment by aggarwalpulkit596** [comment]
You could find it in the test exploration it should be the second last question in that

**Comment by Arjupta** [comment]
### **Earlier**
Screenshot_2020-10-11-12-24-50-42_943a62cb4c6fb83e010e1c2e82766a17

### **Now**
Screenshot_2020-10-11-12-26-27-52_943a62cb4c6fb83e010e1c2e82766a17

Screenshot_2020-10-11-12-27-01-52_943a62cb4c6fb83e010e1c2e82766a17

Instead, both the questions are different but they use the same layout so eligible to compare.
@aggarwalpulkit596 PTAL

**Review by aggarwalpulkit596 — APPROVED** [review]
LGTM

File: app/src/main/res/layout/drag_drop_single_item.xml [hunk 1]
```diff
@@ -10,14 +10,10 @@
 
   <TextView
     android:id="@+id/drag_drop_content_text_view"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:fontFamily="sans-serif"
+    style="@style/Subtitle1"
     tools:text="Hello"
```
Comment (BenHenning) [inline app/src/main/res/layout/drag_drop_single_item.xml]: This tools text shouldn't be here, can you please remove it? I know you didn't add it originally, but this is a good opportunistic fix. :)

**Review by BenHenning — CHANGES_REQUESTED** [review]

**Comment by BenHenning** [comment]
@aggarwalpulkit596 please make sure that there's a follow up action taken after approving (e.g. assigning @rt4914 or I to take a pass before merging, or merging if everything looks good).

File: app/src/main/res/layout/drag_drop_single_item.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/layout/drag_drop_single_item.xml]: Done

**Review by BenHenning — APPROVED** [review]
Thanks @ARJUPTA! LGTM!

**Comment by aggarwalpulkit596** [comment]
give @ARJUPTA  5 points

**Pull request merged**